modules/ROOT/pages/ForUsers/DedicatedServerSetup.adoc

```diff
@@ -87,8 +95,62 @@ Note that applying changes in ficsit-cli is a global action -
 all installations the program is aware of will have any staged changes they may have applied in parallel.
 This does _not_ mean that all installs must be on the same profile.
 
+[id="FileTransferMethods"]
+== File Transfer Methods
+
+=== FTP - File Transfer Protocol
+
+File transfer protocol (FTP) is a common, but outdated method to transfer files over the Internet/Network. 
+The commonly used port number is TCP/21.
+The examples provided are for a self-hosted dedicated server.
+
+The authenticating user requires Read/Write/Delete or Read/Modify permissions.
+
+The path required for FTP to be used is the following:
+`ftp://username:password@ServerNameOrIP:Port/path/`
+
+.Satisfactory Mod Manager FTP Example
+image::DedicatedServers/SMM_FTP.png[Satisfactory Mod Manager Example]
+.Ficsit-CLI FTP Example
+image::DedicatedServers/CLI_FTP.png[Ficsit-CLI Example]
+
+=== SFTP - Secure File Transfer Protocol
+
+Secure File transfer protocol (SFTP) is a common method to transfer files over the Internet/Network. 
+The commonly used port number is TCP/22. 
+The examples provided are for a self-hosted dedicated server.
+
+The authenticating user requires Read/Write/Delete or Read/Modify permissions.
+
+The path required for SFTP to be used is the following:
+`sftp://username:password@ServerNameOrIP:Port/path/`
+
+.Satisfactory Mod Manager SFTP Example
+image::DedicatedServers/SMM_SFTP.png[Satisfactory Mod Manager Example]
+.Ficsit-CLI SFTP Example
+image::DedicatedServers/CLI_SFTP.png[Ficsit-CLI Example]
+
+=== SMB/CIFS - Server Message Block/Common Internet File System _(i.e. Windows Network Share)_ 
+
+[WARNING]
+====
+This feature is currently only supported on ficsit-cli. A future release (soon(TM)) of Satisfactory Mod Manager shall support SMB/CIFS.
+====
+
+Server Message Block (SMB) is also commonly known as Windows File Shares or CIFS (Common Internet File System) is a common method to transfer files over a Network, commonly on Windows Systems, but may be used on Linux/Unix Systems. 
+The commonly used port number is TCP/445.
+The examples provided are for a self-hosted dedicated server.
+
+The authenticating user requires Read/Write/Delete or Read/Modify permissions.
+
+The path required for SMB/CIFS to be used is the following:
+`\\ServerNameOrIP\ShareName\Path`
```

Comment:
For Linux it should be `//ServerNameOrIP/ShareName/Path`, right? So both should be mentioned